Makefile

```diff
@@ -1,5 +1,9 @@
 .DEFAULT_GOAL := generate
 
+# Code Signing Settings
+
+NO_CODE_SIGN_SETTINGS = CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO
```

Comment:
Nice! I was trying to do that in different place https://github.com/lyft/Hammer/pull/56/commits/fa673da7f214bb35b72d9f2306f370ca84b3e905
Thank you!